pkg/config/config.go

```diff
@@ -354,13 +358,16 @@ func load(fs *flag.FlagSet, args []string, loader loaderFunc) (*Config, error) {
 		file            string
 		fileType        string
 		configExpandEnv bool
+		usageStats      bool
 	)
 
 	fs.StringVar(&file, "config.file", "", "configuration file to load")
 	fs.StringVar(&fileType, "config.file.type", "yaml", fmt.Sprintf("Type of file pointed to by -config.file flag. Supported values: %s. %s requires dynamic-config and integrations-next features to be enabled.", strings.Join(fileTypes, ", "), fileTypeDynamic))
 	fs.BoolVar(&printVersion, "version", false, "Print this build's version information")
 	fs.BoolVar(&configExpandEnv, "config.expand-env", false, "Expands ${var} in config according to the values of the environment variables.")
+	fs.BoolVar(&usageStats, "usage-report", false, "When enabled sends usage info of enabled feature flags to Grafana")
```

Comment:
Suuuuper minor nitpick, but how would you feel about the flag being `-report-usage` (as in report usage of flags), instead of `-usage-report` (as in get a usage report)? 🤷